diff --git a/cmake/Modules/Packages/ML-IAP.cmake b/cmake/Modules/Packages/ML-IAP.cmake
index 63f91ba8d3..7b69292045 100644
--- a/cmake/Modules/Packages/ML-IAP.cmake
+++ b/cmake/Modules/Packages/ML-IAP.cmake
@@ -30,7 +30,7 @@ if(MLIAP_ENABLE_PYTHON)
   file(MAKE_DIRECTORY ${MLIAP_BINARY_DIR})
   add_custom_command(OUTPUT  ${MLIAP_BINARY_DIR}/${MLIAP_CYTHON_BASE}.cpp ${MLIAP_BINARY_DIR}/${MLIAP_CYTHON_BASE}.h
           COMMAND            ${CMAKE_COMMAND} -E copy_if_different ${MLIAP_CYTHON_SRC} ${MLIAP_BINARY_DIR}/${MLIAP_CYTHON_BASE}.pyx
-          COMMAND            ${Cythonize_EXECUTABLE} -3 ${MLIAP_BINARY_DIR}/${MLIAP_CYTHON_BASE}.pyx
+                 COMMAND            ${Python_EXECUTABLE} ${Cythonize_EXECUTABLE} -3 ${MLIAP_BINARY_DIR}/${MLIAP_CYTHON_BASE}.pyx
           WORKING_DIRECTORY  ${MLIAP_BINARY_DIR}
           MAIN_DEPENDENCY    ${MLIAP_CYTHON_SRC}
           COMMENT "Generating C++ sources with cythonize...")
