fix liblammps for Mac Os X
author: Jan Janssen (MPIE)
--- python/install.py.orig	2018-12-11 18:39:04.000000000 +0100
+++ python/install.py	2019-02-02 13:21:56.929765000 +0100
@@ -22,6 +22,11 @@
 # copy lammps.py to pydir if it exists
 # if pydir not specified, install in site-packages via distutils setup()
 
+if sys.platform == 'darwin':
+  lib_ext = ".dylib"
+else: 
+  lib_ext = ".so"
+
 if pydir:
   if not os.path.isdir(pydir):
     print( "ERROR: pydir %s does not exist" % pydir)
@@ -36,7 +41,7 @@
   str = "cp ../src/liblammps.so %s" % pydir
   print(str)
   try:
-     shutil.copyfile("../src/liblammps.so", os.path.join(pydir,"liblammps.so") )
+     shutil.copyfile("../src/liblammps" + lib_ext, os.path.join(pydir,"liblammps" + lib_ext) )
   except shutil.Error:
     pass # source and destination are identical
   sys.exit()
@@ -65,7 +70,7 @@
         url = "http://lammps.sandia.gov",
         description = "LAMMPS molecular dynamics library",
         py_modules = ["lammps"],
-        data_files = [(get_python_lib(), ["../src/liblammps.so"])])
+        data_files = [(get_python_lib(), ["../src/liblammps" + lib_ext])])
 except:
   tryuser=True
   print ("Installation into global site-packages dir failed.\nTrying user site dir %s now." % site.USER_SITE)
@@ -81,7 +86,7 @@
     url = "http://lammps.sandia.gov",
     description = "LAMMPS molecular dynamics library",
     py_modules = ["lammps"],
-    data_files = [(site.USER_SITE, ["../src/liblammps.so"])])
+    data_files = [(site.USER_SITE, ["../src/liblammps" + lib_ext])])
   except: 
     print("Installation into user site package dir failed.\nGo to ../python and install manually.")
 
