From 1b4bccc6c7a85224617b4969fb01c732215fbf85 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Thu, 9 Sep 2021 16:28:06 -0600
Subject: [PATCH] Fix mliappy libpython extension

---
 python/lammps/mliap/__init__.py | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/python/lammps/mliap/__init__.py b/python/lammps/mliap/__init__.py
index 0d63cc810b6..62b03635a2f 100644
--- a/python/lammps/mliap/__init__.py
+++ b/python/lammps/mliap/__init__.py
@@ -4,7 +4,11 @@
 # try to improperly start up a new interpreter.
 import sysconfig
 import ctypes
-library = sysconfig.get_config_vars('INSTSONAME')[0]
+
+def conv_ext(lib):
+    return ".".join(lib.split(".")[:-1]) + sysconfig.get_config_var('SHLIB_SUFFIX')
+
+library = conv_ext(lib=sysconfig.get_config_vars('INSTSONAME')[0])
 try:
     pylib = ctypes.CDLL(library)
 except OSError as e:
