From 252c8483c3284f08c1cdcd80935e080f49593224 Mon Sep 17 00:00:00 2001
From: Axel Kohlmeyer <akohlmey@gmail.com>
Date: Thu, 1 Aug 2024 03:45:45 -0400
Subject: [PATCH] make LAMMPS python module compatible with mpi4py 4.x.x

---
 python/lammps/core.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/python/lammps/core.py b/python/lammps/core.py
index 97bcb5157d9..42e8652fef0 100644
--- a/python/lammps/core.py
+++ b/python/lammps/core.py
@@ -342,8 +342,8 @@ def __init__(self,name='',cmdargs=None,ptr=None,comm=None):
     if self.has_mpi_support:
       try:
         from mpi4py import __version__ as mpi4py_version
-        # tested to work with mpi4py versions 2 and 3
-        self.has_mpi4py = mpi4py_version.split('.')[0] in ['2','3']
+        # tested to work with mpi4py versions 2, 3, and 4
+        self.has_mpi4py = mpi4py_version.split('.')[0] in ['2','3','4']
       except ImportError:
         # ignore failing import
         pass
@@ -369,7 +369,7 @@ def __init__(self,name='',cmdargs=None,ptr=None,comm=None):
         if not self.has_mpi_support:
           raise Exception('LAMMPS not compiled with real MPI library')
         if not self.has_mpi4py:
-          raise Exception('Python mpi4py version is not 2 or 3')
+          raise Exception('Python mpi4py version is not 2, 3, or 4')
         if self.MPI._sizeof(self.MPI.Comm) == sizeof(c_int):
           MPI_Comm = c_int
         else:
